posts/views.py

```diff
@@ -1,8 +1,13 @@
 from django.shortcuts import render
+from django.views.generic import ListView
+from .models import Post
 
 
-def feed(request):
-    return render(request, 'posts/feed.html', {'title': 'feed'})
+class PostListView(ListView):
+    model = Post
+    template_name = 'posts/feed.html'
+    context_object_name = 'posts'
+    ordering = ['-date_posted']  # order posts from newest to oldest
```

Comment:
In order to send the `title` you can add this function after line 10:
def get_context_data(self, **kwargs):
    ctx = super(PostListView, self).get_context_data(**kwargs)
    ctx['title'] = 'feed'
    return ctx
it should look like this:
![image](https://user-images.githubusercontent.com/96747472/163358987-ad8e2f74-ca45-4891-80db-4ec3344cffa7.png)

Reply:
Done 👍